### ISSUE TITLE
Error: Unable to access jarfile

### ISSUE THREAD

**Issue by Rurikar22** [Gilmok/U4Rando#1]
One of my favorite games of all times. Super pumped to see this project.

I'm sorta a java newb and when I run the .exe with a rom in the folder I get "Error: Unable to access jarfile" when running it. Any ideas?

**Comment by Gilmok** [comment]
Sorry, looks like I botched the distribution.  I just uploaded a hopefully
working version of the executable.

Thanks for your interest in the project!  More to come soon!

**Comment by Gilmok** [comment]
Is the rando working for you after my latest upload?  I was actually able to reproduce your bug and this latest version was able to work for me.  I'd like to close the issue but I'm not going to do that until I get confirmation that it is working for you.  Thanks.

**Comment by Rurikar22** [comment]
Yup! I played it for a few hours the other night. I need to play the base game to remember some of the main spawn points for stuff haha because the NPCs aren't as helpful right now with items shuffled around. The only issue I came across was the giant fields of grass that made it hard to find some of the towns in my seed (i did a world gen). World gen needs to add more mountains and grass in the bulky land parts still I'd wager.
